processing/src/main/java/io/druid/segment/IndexSpec.java

```diff
@@ -98,7 +99,7 @@ public IndexSpec(
     Preconditions.checkArgument(metricCompression == null || COMPRESSION_NAMES.contains(metricCompression),
                                 "Unknown compression type[%s]", metricCompression);
 
-    this.bitmapSerdeFactory = bitmapSerdeFactory != null ? bitmapSerdeFactory : IndexIO.CONFIGURED_BITMAP_SERDE_FACTORY;
+    this.bitmapSerdeFactory = bitmapSerdeFactory != null ? bitmapSerdeFactory : new ConciseBitmapSerdeFactory();
```

Comment:
Does this mean `druid.processing.bitmap.type` will no longer work to set the default bitmap type?